crates/viewer/re_view_spatial/tests/spawn_heuristics.rs

```diff
@@ -40,6 +40,7 @@ enum EntityKind {
 
 fn build_test_scene(entities: &[(&'static str, EntityKind)]) -> TestContext {
     let mut test_context = TestContext::new_with_view_class::<re_view_spatial::SpatialView2D>();
+    test_context.register_view_class::<re_view_spatial::SpatialView3D>();
```

Comment:
nit: `new_with_view_class` is really just calling `register_view_class`. Would be nicer to read to do

```rs
    let mut test_context = TestContext::new()
    test_context.register_view_class::<re_view_spatial::SpatialView2D>();
    test_context.register_view_class::<re_view_spatial::SpatialView3D>();
```